bin/app.rb

```diff
@@ -34,5 +34,5 @@ def run_service(options)
   puts "Starting Queue Service...".colorize(:cyan)
   app.run_service(options)
 rescue ArgumentError => e
-  puts "There is an exception in the #{service} service #{e.inspect}".colorize(:red)
+  QIt.log.error "There is an exception in the #{service} service #{e.inspect}"
```

Comment:
Currently you have used `QIt.log.error` - this means that it is tightly coupled with the `QIt` application. Going ahead - have `App` or something generic that would be able to use `App.log.error` as a means to log errors. This could be applicable across applications that you generate using the same framework. And there would be a lesser learning curve.